app/assets/stylesheets/bootstrap_overrides.scss

```diff
@@ -88,6 +88,12 @@ a {
   font-weight: bold;
   background-color: inherit;
 }
+.navbar-default .navbar-nav > .open > a,
+.navbar-default .navbar-nav > .open > a:hover,
+.navbar-default .navbar-nav > .open > a:focus {
```

Comment:
Could you explain this more to me later? I was trying to override the Bootstrap defaults and that's how they wrote it so I copied and pasted and then changed attributes.